src/interfaces/protocols/handlers/protocols-query.ts

```diff
@@ -32,11 +32,14 @@ export class ProtocolsQueryHandler implements MethodHandler {
       return MessageReply.fromError(e, 401);
     }
 
-    const query = {
+    const query: Filter = {
       interface : DwnInterfaceName.Protocols,
       method    : DwnMethodName.Configure,
-      ...message.descriptor.filter
```

Comment:
NIT: i recognize that there's no need for this anymore since the only value in `filter` is `protocol`, but some part of me feels like we should still leave this here just for future proofing sake 🤔

Reply:
added back in